apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx

```diff
@@ -3,26 +3,13 @@ import { ISiteDefinition, LoadingComponent } from '@uifabric/example-app-base/li
+  siteLogoSource: 'https://static2.sharepointonline.com/files/fabric/fabric-website/images/microsoftuifabric-logo-rgb_no-padding.svg',
```

Comment:
Much better! :) You may also need to remove some styling I added to make the hack-logo look right--unfortunately I don't remember exactly where the styling was added...